src/apps/api/urls.py

```diff
@@ -38,6 +38,7 @@
     path('upload_submission_scores/<int:submission_pk>/', submissions.upload_submission_scores),
     path('can_make_submission/<phase_id>/', submissions.can_make_submission, name="can_make_submission"),
     path('add_submission_to_leaderboard/<int:submission_pk>/', leaderboards.add_submission_to_leaderboard, name='add_submission_to_leaderboard'),
+    path('remove_submission_from_leaderboard/<int:submission_pk>/', leaderboards.remove_submission_from_leaderboard, name='remove_submission_from_leaderboard'),
```

Comment:
Let's consolidate these two endpoints into a single one, `submission_leaderboard_connection` to which we can either `POST` or `DELETE`.

Reply:
Done